src/components/Table.js

```diff
@@ -0,0 +1,48 @@
+import React, { useState, useEffect } from 'react';
+import './Table.css';
+
+export default function Table() {
+  const [items, setItems] = useState([]);
+
+  useEffect(() => {
+    getData();
+  });
+
+  async function getData() {
```

Comment:
If you are going with functional component approach try making custom hook for each functionality/api